interpreter/cling/lib/Interpreter/DefinitionShadower.cpp

```diff
@@ -94,9 +98,12 @@ namespace cling {
   }
 
   void DefinitionShadower::invalidatePreviousDefinitions(NamedDecl *D) const {
+    // NotForRedeclaration: lookup anything visible; follows using directives
     LookupResult Previous(*m_Sema, D->getDeclName(), D->getLocation(),
-                   Sema::LookupOrdinaryName, Sema::ForVisibleRedeclaration);
+                   Sema::LookupOrdinaryName, Sema::NotForRedeclaration);
+    Previous.suppressDiagnostics();
     m_Sema->LookupQualifiedName(Previous, m_TU);
+    bool shadowsDeclInStd{};
```

Comment:
Could you use the more explicit version, please?
```suggestion
    bool shadowsDeclInStd = false;
```